osmosis/src/storeProvider.js

```diff
@@ -1,16 +1,19 @@
 /**
  * @param {Object[]} storeProviders
- * @param {Object} componentToWrap 
+ * @param {Object} componentToWrap
  * @returns {Object} wrappedComponent
  */
 export const StoreProvider = (storeProviders, wrappedComponent) => {
-  if(!Array.isArray(storeProviders)) throw new Error('StoreProvider requires an array of wrapper functions')
-  if(!wrappedComponent) throw new Error('StoreProvider requires a component to wrap')
+  if (!Array.isArray(storeProviders)) throw new Error('StoreProvider requires an array of wrapper functions');
+  if (!wrappedComponent) throw new Error('StoreProvider requires a component to wrap');
 
   try {
-    storeProviders.reverse().forEach(provider => (wrappedComponent = provider(wrappedComponent)));
+    storeProviders.reverse().forEach(provider => {
```

Comment:
Purely curious, do you recall any specific reason we use forEach here instead of map?

Just wondering about the guts of osmosis as I look at each PR.

Reply:
I don't think you want a map here as that would return an array of items when what we really want it just a nested/wrapped single item. You could argue that this could be a `reduce`, but honestly in this case IMO there's not much actual difference

Reply:
Oops, yep, reduce. Thank you for the explanation 🙏🏼